projects/layout/src/modules/text-expand-repeater/text-expand-repeater.component.scss

```diff
@@ -8,3 +8,11 @@
   white-space: nowrap;
   margin-left: 40px;
 }
+
+.sky-text-expand-repeater-bullet-style-none {
+  list-style-type: none;
+}
+
+.sky-text-expand-repeater-see-more-bullet-style-none {
+  margin-left: 0;
```

Comment:
Went back and forth on this a little bit. This will need done when the theme changes come into play, but looks a little goofy without. If this is going to be pushed out and publicized before the theme change, then I can back this style out (or comment it out for later)